#いくつの未訪問グリッドセルを内包しているか
uint32 size
#ロボットの現在位置から一番近いグリッドセルまでの距離
float64 min_distance
#目標地点
geometry_msgs/Point travel_point
